git-branchless/tests/test_hooks.rs

```diff
@@ -385,3 +388,54 @@ fn test_git_am_recorded() -> eyre::Result<()> {
 
     Ok(())
 }
+
+#[cfg(unix)]
+#[test]
+fn test_git_rebase_multiple_fixup_does_not_strand_commits() -> eyre::Result<()> {
+    let git = make_git()?;
+    git.init_repo()?;
+
+    git.detach_head()?;
+    git.commit_file_with_contents_and_message_and_file_name(
+        "test1",
+        1,
+        "bleh",
+        "create",
+        "test1.txt",
+    )?;
+    git.commit_file_with_contents_and_message_and_file_name(
+        "test2",
+        2,
+        "bleh",
+        "fixup! create",
+        "test1.txt",
+    )?;
+    git.commit_file_with_contents_and_message_and_file_name(
+        "test3",
+        3,
+        "bleh",
+        "fixup! create",
+        "test1.txt",
+    )?;
+
+    run_in_pty_with_command(
+        &git,
+        &["rebase"],
+        &["-i", "--autosquash", "master"],
+        &[
+            PtyAction::WaitUntilContains(" "),
+            PtyAction::Write(CARRIAGE_RETURN),
+        ],
+    )?;
```

Comment:
Can we just write `git.run(&["rebase", "--autosquash", "master"])`? It seems to work in my testing. If so, do that and revert the changes to `run_in_pty`.